app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CanTheyUseOurServiceController.scala

```diff
@@ -120,36 +130,91 @@ class CanTheyUseOurServiceController @Inject()(
-            Redirect(routes.CanTheyUseOurServiceController.whoIsIndividualRepresenting())
-          } else {
-            numberOfPropertiesForm
-              .bindFromRequest()
-              .fold(
-                formWithErrors => BadRequest(howManyPropertiesPage(formWithErrors)), { numberOfProperties =>
-                  val newSubscribed = subscribed.copy(
-                    individualTriageAnswers = Some(
-                      individualTriageAnswers.copy(numberOfProperties = Some(numberOfProperties)))
-                    )
-
-                  updateSession(sessionStore, request)(_.copy(journeyStatus = Some(newSubscribed)))
-                    .map {
-                      case Left(e) =>
-                        logger.warn("Could not update session", e)
-                        errorHandler.errorResult()
-
-                      case Right(_) =>
-                        Ok(s"Got number of properties $numberOfProperties")
+  def howManyPropertiesSubmit(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withIndividualTriageAnswers(request) {
+      case (_, subscribed, individualTriageAnswers) =>
+        if (individualTriageAnswers.individualUserType.isEmpty) {
+          Redirect(routes.CanTheyUseOurServiceController.whoIsIndividualRepresenting())
+        } else {
+          numberOfPropertiesForm
+            .bindFromRequest()
+            .fold(
+              formWithErrors => BadRequest(howManyPropertiesPage(formWithErrors)), { numberOfProperties =>
+                val newSubscribed = subscribed.copy(
+                  individualTriageAnswers =
+                    Some(individualTriageAnswers.copy(numberOfProperties = Some(numberOfProperties)))
+                )
+
+                updateSession(sessionStore, request)(_.copy(journeyStatus = Some(newSubscribed)))
+                  .map {
+                    case Left(e) =>
+                      logger.warn("Could not update session", e)
+                      errorHandler.errorResult()
+
+                    case Right(_) =>
+                      numberOfProperties match {
+                        case NumberOfProperties.One =>
+                        Redirect(routes.CanTheyUseOurServiceController.whenWasDisposalDate())
+
+                        case NumberOfProperties.MoreThanOne =>
+                          Ok("multiple disposals not handled yet")
+                      }
+                  }
+
+              }
+            )
+        }
+    }
+  }
+
+  def whenWasDisposalDate(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withIndividualTriageAnswers(request) {
+      case (_, subscribed, individualTriageAnswers) =>
+        if (individualTriageAnswers.numberOfProperties.isEmpty) {
+          Redirect(routes.CanTheyUseOurServiceController.howManyProperties())
+        } else {
+          val today = LocalDate.now(clock)
```

Comment:
Better to use `Instant` to represent a moment in time on the UTC timeline.

Reply:
we don't need/have any time information so that part of `Instant` is wasted on us a bit. We only ever use the date as a day, month, year combo so I'm not sure what more `Instant`  would give us

Reply:
I'd prefer something like this as it makes it clear what's going on and dare I say, more idiomatic in terms of the usage os the Java time API:
This is what I meant but leave it as is. This encoding, for me, makes it clear what the intent is but YMMV.

```scala
Instant instantOfNow = Instant.now();
LocalDate localDate = LocalDateTime.ofInstant(instantOfNow, ZoneOffset.UTC).toLocalDate();
```